src/main/groovy/org/cassalog/core/CreateKeyspace.groovy

```diff
@@ -72,7 +72,7 @@ CREATE KEYSPACE $name WITH replication = { 'class': '${replication.strategy}', '
 
   void replication(Closure closure) {
     closure.delegate = replication
-    closure.resolveStrategy = Closure.DELEGATE_ONLY
+    closure.resolveStrategy = Closure.OWNER_FIRST
```

Comment:
so, I'm not an expert either, but since the strategy is resolve from DELEGATE_ONLY and the delegate is replication, that results on assigning to replication the exact same value that replication has, which in the end resulted on my overriden properties not being honoured